edge/services/mms/README.md

```diff
@@ -0,0 +1,149 @@
+# Horizon Model Management Service (MMS) Example
+
+## Getting Ready
+
+- It is assumed you have gone through the developer workflow for at least one of the other Horizon examples on this host, so you have verified your credentials are configured in the environment of this shell, you are logged-in to DockerHub, and you have created your cryptographic signing key pair, etc.
+
+- Clone this git repo:
+
+```
+git clone git@github.com:open-horizon/examples.git
+```
+
+- Enter the `mms` example directory
+
+```
+cd  examples/edge/services/mms
+```
+
+- Set the values in `horizon/hzn.json` to your own values.
+- Add those values into your shell environment.
+
+```
+`hzn util configconv -f horizon/hzn.json`
+```
+
+- Build and appropriately tag this example Docker container
+
+```
+docker build -t "${DOCKER_IMAGE_BASE}_$(hzn architecture):${SERVICE_VERSION}" .
+```
+
+## Running In Development Mode
+
+- Use the developer tool to run the container with a local development instance of the Cloud Sync Server (CSS). Normally, in production, you will use the CSS in the IBM Cloud, but during development it is convenient to have a dedicated and private "dev CSS" instance you can use. So we will show that approach here first.
```

Comment:
Instead of "Cloud Sync Server", I would rather that you used the term "Model Management Service". You could add that the Cloud Sync Service (CSS) container implements the cloud side of the MMS.

Also, they might be deployed in ICP so refering to the IBM cloud might be problematic, not sure what to say instead.

Reply:
I removed all ESS and CSS and use only MMS in the README now. Also addressed ICP.